plugins/pdr_deterministic_plugin/ufm_sim_web_service/isolation_mgr.py

```diff
@@ -259,76 +274,122 @@ def check_link_down_condition(self, port_obj, port_name, ports_counters):
             return Issue(port_name, Constants.ISSUE_LINK_DOWN)
         return None
     
+    def calc_error_rate(self, port_obj, row, timestamp):
+        """
+        Calculate the error rate of the port and update the counters_values
+        """
+        rcv_error = get_counter(Constants.RCV_ERRORS_COUNTER, row)
+        rcv_remote_phy_error = get_counter(Constants.RCV_REMOTE_PHY_ERROR_COUNTER, row)
+        errors = rcv_error + rcv_remote_phy_error
+        error_rate = self.get_rate_and_update(port_obj, Constants.ERRORS_COUNTER, errors, timestamp)
+        return error_rate        
+
+    def check_pdr_issue(self, port_obj, port_name, row, timestamp):
+        """
+        Check if the port passed the PacketDropRate threshold and return an issue
+        """
+        rcv_pkts = get_counter(Constants.RCV_PACKETS_COUNTER, row)
+        rcv_pkt_rate = self.get_rate_and_update(port_obj, Constants.RCV_PACKETS_COUNTER, rcv_pkts, timestamp)
+        error_rate = self.calc_error_rate(port_obj, row, timestamp)
+        if rcv_pkt_rate and error_rate / rcv_pkt_rate > self.max_pdr:
+            return Issue(port_name, Constants.ISSUE_PDR)
+        return None
+    
+    def check_temp_issue(self, port_obj, port_name, row, timestamp):
```

Comment:
port name and port obje ... no need port name

Reply:
fixed